packages/abi-typegen/README.md

```diff
@@ -0,0 +1,89 @@
+# `@fuel-ts/abi-typegen`
+
+Generate TypeScript bindings for [Sway](https://github.com/fuellabs/sway) smart contracts.
+
+See the full ABI-spec [here](https://github.com/FuelLabs/fuel-specs/blob/master/src/protocol/abi/json_abi_format.md).
+
+# Table of contents
+
+- [Documentation](#documentation)
+- [Usage](#usage)
+  - [Installation](#installation)
+  - [Full SDK Installation](#full-sdk-installation)
+- [Type's Conversion Table](#types-conversion-table)
+- [Contributing](#contributing)
+- [Changelog](#changelog)
+- [License](#license)
+
+## Documentation
+
+See [Fuel-ts Documentation](https://fuellabs.github.io/fuels-ts/packages/fuel-ts-abi-typegen/)
+
+## Usage
+
+### Installation
+
+```sh
+yarn add @fuel-ts/abi-typegen
+# or
+npm add @fuel-ts/abi-typegen
+```
+
+### Generating types
+
+```sh
+npx typegen -i ./out/debug/*-abi.json -o ./src/contracts
```

Comment:
Should have fuels on the command?
```suggestion
npx fuels typegen -i ./out/debug/*-abi.json -o ./src/contracts
```

Reply:
No, only when choosing the full SDK installation.

Reply:
At this point, the `fuels` prefix is required only when using the umbrella package.

I will, however, rename it from `typegen` to `fuels-typegen` (_only for the standalone version).

> _Same approach I used for the `fuels version` x `fuels-version` command from #616_.